contrib/platform/test/com/sun/jna/platform/win32/Kernel32UtilTest.java

```diff
@@ -253,4 +256,34 @@ public final void testWritePrivateProfileSection() throws IOException {
             reader.close();
         }        
     }
+    
+    public void testGetResource() {
+		// On Windows 7, "14" is the type assigned to the "My Computer" icon
+		// (which is named "ICO_MYCOMPUTER")
+    	byte[] results = Kernel32Util.getResource("c:\\windows\\explorer.exe", "14", "ICO_MYCOMPUTER");
```

Comment:
I don't think this will work - I think you can use _Kernel32.INSTANCE.GetEnvironmentVariable("WINDIR")_ and then create the path yourself - e.g.:
```java
String winDir = Kernel32.INSTANCE.GetEnvironmentVariable("WINDIR");
...handle error if any...
String explorerDir = winDir + File.separator + "explorer.exe";
// Should check if 'winDir' already ends in File.separatorChar and avoid appending it again if so
...run your test...
```